Mathlib/Data/Matroid/Rank/ENat.lean

```diff
@@ -298,6 +261,170 @@ lemma eRk_inter_add_eRk_union_le (M : Matroid α) (X Y : Set α) :
 
 alias eRk_submod := eRk_inter_add_eRk_union_le
 
+lemma eRk_submod_insert (M : Matroid α) (X Y : Set α) :
+    M.eRk (insert e (X ∩ Y)) + M.eRk (insert e (X ∪ Y))
+      ≤ M.eRk (insert e X) + M.eRk (insert e Y) := by
```

Comment:
I'm not sure how I feel about the naming of these ones - `submod` is an abbreviation for the meaning, but `eRk_inter_add_eRk_union_le` is closer to the symbol-reading convention. And for the new ones, having an alias on each seems wasteful... What's your take?